java/src/jmri/jmrix/cmri/serial/nodeconfigmanager/NodeConfigManagerFrame.java

```diff
@@ -1333,7 +1328,7 @@ public void addNodeButtonActionPerformed() {
         // reset text displays after succefully adding node
         resetNotes();
         changedNode = true;
-
+        log.info("changedNode = "+changedNode);
         // provide user feedback
         statusText1.setText(Bundle.getMessage("FeedBackAdd") + " " + Integer.toString(nodeAddress));
```

Comment:
Another comment for the future:  You can allow people more flexibility to translate/format the message by letter the Bundle code do the combination:  Change that line to make the integer an argument to Bundle:
```
statusText1.setText(Bundle.getMessage("FeedBackAdd", Integer.toString(nodeAddress)));
```
and have the FeedBackAdd property use {}  specify where to include it:
```
FeedBackAdd         = C/MRI Node added. Node Address = {}
```

Reply:
Will do that.